st2tests/tests/config.schema.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
The tests pack I'm asking you to modify is located @ https://github.com/StackStorm/st2tests/tree/master/packs/tests.  It is a separate repo.  You're adding files in the wrong place.